### PR TITLE
Bug fix printing from onboard + cleanup

### PR DESCRIPTION
**Pull request by digant73**
**BUGFIXES:**
* Fixed a bug introduced by #2599: The matching rule on parseAck.c became a sink once a print from (remote) onboard is started and remained a sink until the media is changed (e.g. a print from SD card is started). This bug did not allow the parsing of any command (e.g. sent by Terminal or remote host) whose matching rule followed the sink.

**IMPROVEMENTS:**
* Some code reduction
* Some cleanup + added comments

**PR STATE:** Ready for merge

### CODE REVIEW

File: TFT/src/User/Menu/Popup.h [hunk 1]
```diff
@@ -96,13 +95,12 @@ void loopPopup(void);
...
  * @param msg the body of the message/reminder to be displayed
  */
-#define popupSplash(_type, _title, _msg)                  \
-  {                                                         \
+#define popupSplash(_type, _title, _msg)                 \
+  {                                                      \
     setDialogText(_title, _msg, LABEL_NULL, LABEL_NULL); \
-    showDialog(_type, NULL, NULL, NULL);                    \
+    showDialog(_type, NULL, NULL, NULL);                 \
   }
```
Comment (kisslorand) [inline TFT/src/User/Menu/Popup.h:102]: Copy & paste from PR #2603

File: TFT/src/User/Menu/Popup.h [hunk 2]
```diff
@@ -50,9 +50,8 @@ void _setDialogCancelTextLabel(int16_t index);
     setDialogCancelText(canceltext);                  \
   }
 
-void popupDrawPage(DIALOG_TYPE type, BUTTON * btn, const uint8_t * title, const uint8_t * context, const uint8_t * yes,
-                    const uint8_t * no);
-//void popupReminder(DIALOG_TYPE type, uint8_t* title, uint8_t* msg);
```
Comment (kisslorand) [inline TFT/src/User/Menu/Popup.h:55]: "Borrowed" from PR #2607

File: TFT/src/User/my_misc.h [hunk 3]
```diff
@@ -34,17 +34,17 @@ extern "C" {
 // Get bit status at selected index
 #define GET_BIT(num, index) ((num >> index) & 1)
 
-// set bit status at selected index to 1
+// Set bit status at selected index to 1
```
Comment (kisslorand) [inline TFT/src/User/my_misc.h:37]: Copy & paste from PR #2601

File: TFT/src/User/my_misc.h [hunk 4]
```diff
@@ -34,17 +34,17 @@ extern "C" {
...
 #define SET_BIT_ON(num, index) num = (1 << index) | num
 
-// set bit status at selected index to 0
+// Set bit status at selected index to 0
```
Comment (kisslorand) [inline TFT/src/User/my_misc.h:40]: Copy & paste from PR https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/pull/2601

File: TFT/src/User/my_misc.h [hunk 5]
```diff
@@ -34,17 +34,17 @@ extern "C" {
...
 #define SET_BIT_OFF(num, index) num = num & (~(1 << index))
 
-// set bit status value (0 or 1) at selected index
+// Set bit status value (0 or 1) at selected index
```
Comment (kisslorand) [inline TFT/src/User/my_misc.h:43]: Copy & paste from PR https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/pull/2601

File: TFT/src/User/my_misc.h [hunk 6]
```diff
@@ -34,17 +34,17 @@ extern "C" {
...
 #define SET_BIT_VALUE(num, index, value) num = ((value) == 1) ? (1 << index) | num : num & (~(1 << index))
 
 // Toggle bit status at selected index
-#define TOGGLE_BIT(num, index) num =  num ^ (1 << index)
+#define TOGGLE_BIT(num, index) num = num ^ (1 << index)
```
Comment (kisslorand) [inline TFT/src/User/my_misc.h:47]: "Borrowed" from PR https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/pull/2601

**Comment by digant73** [comment]
@kisslorand Please, have a break with such kind of meaningless complaints. They contribute only to damage the project. Thanks.
@bigtreetech If kisslorand's complaints are acceptable, please, when possible, review and hopefully merge the dated #2603, source of almost all the complaints apparently so important now to kisslorand. Otherwise, please do not take them into account for this and further PRs. I have no time to waste with such kind of clear excuses/blackmails having nothing to do with the real content of the PRs.

**Comment by kisslorand** [comment]
@bigtreetech
Please take a look also at this: #2685. Thank you!

**Comment by digant73** [comment]
Closing just to avoid any infinite loop. This PR will be eventually submitted/adapted in the future once kisslorand's constraints will be satisfied or rejected

**Pull request closed without merging**